core/src/test/scala/cats/derived/traverse.scala

```diff
@@ -0,0 +1,163 @@
+package cats
+package derived
+
+import cats.derived.TestDefns._
+import cats.implicits._
+import cats.laws.discipline.{FoldableTests, TraverseTests}
+import org.scalacheck.Test
+import org.scalatest.FreeSpec
+import shapeless.test.illTyped
+
+class TraverseSuite extends FreeSpec {
```

Comment:
Nitpick: is there any technical reason that this test uses `FreeSpec` vs. the `KittenSuite` based of `FunSpec`?

Reply:
I actually copy pasted the `functor` test suite and adapted it to `traverse`